components/permalink_view/permalink_view.jsx

```diff
@@ -32,6 +32,7 @@ export default class PermalinkView extends React.PureComponent {
         actions: PropTypes.shape({
             focusPost: PropTypes.func.isRequired,
         }).isRequired,
+        currentUserId: PropTypes.string.isRequired,
```

Comment:
Why is this being added? It doesn't look like this is used anywhere

Reply:
Yep, missed passing it into `focusPost`.